tests/test_maintel_enable_mtcs.py

```diff
@@ -19,79 +19,19 @@
 # You should have received a copy of the GNU General Public License
 # along with this program. If not, see <https://www.gnu.org/licenses/>.
 
-import logging
-import random
 import unittest
 
-from lsst.ts import salobj, standardscripts
-from lsst.ts.observatory.control.mock import MTCSMock
+from lsst.ts import standardscripts
 from lsst.ts.standardscripts.maintel import EnableMTCS
 
-random.seed(47)  # for set_random_lsst_dds_partition_prefix
-
-logging.basicConfig()
-
 
 class TestEnableMTCS(
     standardscripts.BaseScriptTestCase, unittest.IsolatedAsyncioTestCase
 ):
     async def basic_make_script(self, index):
         self.script = EnableMTCS(index=index)
-        self.mtcs_mock = MTCSMock()
-
-        return (self.script, self.mtcs_mock)
-
-    async def test_components(self):
-        async with self.make_script():
-            for component in self.script.group.components_attr:
-                with self.subTest(f"Check {component}", component=component):
-                    if getattr(self.script.group.check, component):
-                        assert component in self.script.components()
-
-    async def test_run(self):
```

Comment:
All the `Enable` scripts are a subclass of `EnableGroup`, and all this is already tested on other scripts. I removed these because they are duplicated tests and to make the tests go quicker.

Reply:
A okay, that makes sense. Can you add a one line comment to the .misc.rst news fragment to cover that?